static/app/components/organizations/environmentPageFilter/index.spec.tsx

```diff
@@ -118,19 +118,11 @@ describe('EnvironmentPageFilter', function () {
     );
 
     // Open menu again, click "Clear"
+    await userEvent.click(screen.getByRole('button', {name: 'prod, stage'}));
     await userEvent.click(screen.getByRole('button', {name: 'Clear'}));
 
     // Trigger button was updated
     expect(screen.getByRole('button', {name: 'All Envs'})).toBeInTheDocument();
-
-    // Router was updated with new empty value
-    waitFor(() =>
-      expect(router.push).toHaveBeenCalledWith(
-        expect.objectContaining({
-          query: {environment: []},
-        })
-      )
-    );
```

Comment:
`router.push` wasn't called because there's no difference between the old and new queries — both are `{}`. (when we selected `prod` & `stage` above, so actual query wasn't updated, it remained empty `{}`).